lib/Utils/Backends.pm

```diff
@@ -39,6 +39,8 @@ use constant {
           is_spvm
           is_pvm_hmc
           is_generalhw
+          is_ipxe_boot
+          is_uefi_boot
```

Comment:
Are they suitable to be put in backends.pm? Not quite backend related...

Reply:
Do you have better place to hold them ? If no, I think this is best place.